src/blockchain/consensus.md

```diff
@@ -0,0 +1,36 @@
+# Consensus
+
+Each time you send a transaction to Iroha, it gets put into a queue. When
+it's time to produce a new block, the queue is emptied, and the consensus
```

Comment:
I guess the queue is not emptied but some elements of that are taken to form a block (potentially; positive scenario), there is a configurable limits of a block size, so in general case queue might contain lots of transactions and those won't fit in a single block